packages/extension/src/manifest.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "https://json.schemastore.org/chrome-manifest",
```

Comment:
I'm a bit hesitant to make any changes to the manifest.
We don't know the full impact and it seems to be working fully so far.

Reply:
The result is what matters. Let's try to build it and see if the `$schema` field gets dropped out.
If it gets dropped, then we have nothing to worry about. If it remains, let's think what to do

Reply:
What would be the downside if the `$schema` field does not drop?

Reply:
Manifest is something that gets reviewed very carefully by the stores. They are very sensitive to changes there, which can delay the review by days. We can't keep it in the final version of the manifest. For development version that's fine of course

Reply:
ohh, ok. lets not complicate things then. I am closing this PR for now since it is not something super important